src/main/scala/nl/knaw/dans/api/sword2/CollectionListManagerImpl.scala

```diff
@@ -28,7 +28,7 @@ class CollectionListManagerImpl extends CollectionListManager {
   @throws(classOf[SwordError])
   def listCollectionContents(collectionIRI: IRI, auth: AuthCredentials, config: SwordConfiguration): Feed = {
     implicit val settings = config.asInstanceOf[SwordConfig].settings
-    Authentication.checkAuthentication(auth)
+    Authentication.checkAuthentication(auth).get
```

Comment:
Since `SwordID.extract` returns a `Try` as well, it can be done in a for-comprehension!